index.html

```diff
@@ -43,104 +53,131 @@ <h1>Hey there. I'm <br> Hiwot<span class="h1color"><br>
                     <h4>Multi-Post Stories Gain+Glory</h4>
-                    <ul class="button-detail"> 
+                    <ul class="button-detail" id="desktop-button-detail"> 
                         <li>Ruby on rails</li>
-                         <li>css</li>
+                        <li>css</li>
                         <li>JavScript</li>
                         <li>html</li>
                     </ul>
-                     <button class="see-project">See project </button>
-                 </div> 
+                    <button class="see-project" id="desktop-see-project">See project </button>
+                </div>  
             </div>
-            <div class="project4">  
-                 <div class="project-detail">
+            <div class="project4" id="desktop-project4">  
+                <div class="project-detail" id="desktop-project-detail">
                     <h4>Multi-Post Stories Gain+Glory</h4>
-                    <ul class="button-detail"> 
+                    <ul class="button-detail" id="desktop-button-detail"> 
                         <li>Ruby on rails</li>
                         <li>css</li>
                         <li>JavScript</li>
                         <li>html</li>
                     </ul>
-                    <button class="see-project">See project </button>
+                    <button class="see-project" id="desktop-see-project">See project </button>
                 </div> 
             </div>
-             <div class="project5">  
-                <div class="project-detail">
+             <div class="project5" id="desktop-project5">  
+                <div class="project-detail" id="desktop-project-detail">
                     <h4>Multi-Post Stories Gain+Glory</h4>
-                    <ul class="button-detail"> 
+                    <ul class="button-detail" id="desktop-button-detail"> 
                         <li>Ruby on rails</li>
                         <li>css</li>
                         <li>JavScript</li>
                         <li>html</li>
                     </ul>
-                    <button class="see-project"> <a href="https://github.com/hiwina"></a> See project </button>
+                    <button class="see-project" id="desktop-see-project">See project </button>
                 </div> 
             </div>
-            <div class="project6">  
-                <div class="project-detail">
+            <div class="project6" id="desktop-project6">   
+                <div class="project-detail" id="desktop-project-detail">
                     <h4>Multi-Post Stories Gain+Glory</h4>
-                    <ul class="button-detail"> 
+                    <ul class="button-detail" id="desktop-button-detail"> 
                         <li>Ruby on rails</li>
                         <li>css</li>
                         <li>JavScript</li>
                         <li>html</li>
                     </ul>
-                    <button class="see-project">See project </button>
```

Comment:
- [ ] Please kindly follow spacing(padding , margin) in order to stick to the figma design. its good for your future job.
### Your design👇
![padding222](https://user-images.githubusercontent.com/66992901/199487574-15985c08-fdd0-4510-bc3c-1a4dcb0b6574.PNG)
### Figma design👇
![paddfig](https://user-images.githubusercontent.com/66992901/199487885-3cd43f05-82a7-4b18-abc0-dedcc07e548a.PNG)